Property_map/include/CGAL/Dynamic_property_map.h

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2007  GeometryFactory (France).  All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org); you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public License as
+// published by the Free Software Foundation; either version 3 of the License,
+// or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+//
+//
+// Author(s)     : Andreas Fabri
+
+#ifndef CGAL_DYNAMIC_PROPERTY_MAP_H
+#define CGAL_DYNAMIC_PROPERTY_MAP_H
+
+#include <CGAL/boost/graph/properties.h>
+#include <CGAL/property_map.h>
+#include <boost/shared_ptr.hpp>
+#include <boost/unordered_map.hpp>
+
+namespace CGAL {
+
+namespace internal {
+
+template <typename K, typename V>
+struct Dynamic_property_map {
+
+  typedef K key_type;
+  typedef V value_type;
+  typedef value_type& reference;
```

Comment:
`reference` is the return type of `get` so it must be `const value_type&` and `get` function should be updated